frameworks/kafka/versions.sh

```diff
@@ -1,3 +1,15 @@
 #!/usr/bin/env bash
 
-export TEMPLATE_KAFKA_VERSION="2.12-1.1.0"
+# This `TEMPLATE_KAFKA_VERSION` string is composed of:
+# <scala-version>-<kafka-version>
+
+# The available versions of Kafka (with the Scala versions used) can be found
+# at:
+# https://kafka.apache.org/downloads.
+
+# The version here is looked for on the Mesosphere Amazon S3 downloads
+# repository.
+# To upload a new version of the Kafka package to that repository, follow the
+# instructions at:
+# https://wiki.mesosphere.com/display/ENGINEERING/Uploading+an+asset+to+production
```

Comment:
A newly-released package's assets aren't uploaded using those Jenkins jobs. The assets are built and uploaded throughout the [release process](https://wiki.mesosphere.com/display/ENGINEERING/Releasing+a+Framework).

Reply:
Sorry for the misunderstanding, after chatting with @mhrabovcin I realized that by "Kafka package" you meant the Kafka base-tech release artifact. Words are hard!